nucleus/core/kernel/src/main/java/org/glassfish/kernel/KernelLoggerInfo.java

```diff
@@ -94,206 +94,206 @@ public static Logger getLogger() {
-    
+
     @LogMessageInfo(
             message = "Cannot process admin console request.",
             cause = "InterruptedException occurred while the service thread is running.",
             action = "Check the system logs and contact Payara Support.",
             level = "SEVERE")
     public static final String consoleCannotProcess = LOGMSG_PREFIX + "-00008";
-    
+
     @LogMessageInfo(
             message = "Unable to serve resource: {0}. Cause: {1}",
             cause = "An I/O error occurred while serving a resource request.",
             action = "Check the system logs and contact Payara Support.",
             level = "SEVERE")
     public static final String consoleResourceError = LOGMSG_PREFIX + "-00009";
-    
+
     @LogMessageInfo(
             message = "Resource not found: {0}",
             level = "WARNING")
     public static final String consoleResourceNotFound = LOGMSG_PREFIX + "-00010";
-    
+
     @LogMessageInfo(
             message = "Console cannot be initialized due to an exception.",
             level = "INFO")
     public static final String consoleCannotInitialize = LOGMSG_PREFIX + "-00011";
-    
+
     @LogMessageInfo(
             message = "Cannot write property ''{0} = {1}'' for AdminService in domain.xml, exception: {2}",
             level = "INFO")
     public static final String consoleCannotWriteProperty = LOGMSG_PREFIX + "-00012";
-    
+
     @LogMessageInfo(
             message = "Shutdown procedure finished",
             level = "INFO")
     public static final String shutdownFinished = LOGMSG_PREFIX + "-00013";
-    
+
     @LogMessageInfo(
             message = "Shutdown required",
             cause = "An unexpected exception occurred while changing run levels.  A shutdown is required.",
             action = "Check the system logs and contact Payara Support.",
             level = "SEVERE")
     public static final String shutdownRequired = LOGMSG_PREFIX + "-00014";
-    
+
     @LogMessageInfo(
             message = "Shutdown requested",
             level = "INFO")
     public static final String shutdownRequested = LOGMSG_PREFIX + "-00015";
-    
+
     @LogMessageInfo(
             message = "Startup service failed to start",
             cause = "An unexpected exception occurred while starting the startup service.  A shutdown is required.",
             action = "Check the system logs and contact Payara Support.",
             level = "SEVERE")
     public static final String startupFailure = LOGMSG_PREFIX + "-00016";
-    
+
     @LogMessageInfo(
             message = "{0} ({1}) startup time : {2} ({3}ms), startup services({4}ms), total({5}ms)",
             level = "INFO")
     public static final String startupEndMessage = LOGMSG_PREFIX + "-00017";
-    
+
     @LogMessageInfo(
-            message = "TOTAL TIME INCLUDING CLI: {0}",
+            message = "Total startup time including CLI: {0} ms",
             level = "INFO")
```

Comment:
Shouldn't this correlate with level used in previous file, i. e. `CONFIG`?

Reply:
I have no idea why LogMessageInfo has the level attribute and how it is used. If it is meaningless, I don't care ... otherwise I would have to review all LogMessageInfo usages, I expect it differs in much more cases ...